rfd/0061-tsh-aliases.md

```diff
@@ -138,23 +168,23 @@ using `tsh connect leaf ubuntu@node-1` command:
 
 ```yaml
 aliases:
-    "connect": "bash -c '$TSH login $1 && $TSH ssh $2'"
+  "connect": "exec -- -c '$TSH login $0 && $TSH ssh $1'"
```

Comment:
I wonder why can't we just inject our env vars in all aliases always instead of requiring exec. One less thing users have to learn.

Reply:
Aliases may point to existing commands, in which case the notion of env variables doesn't really apply?